front/src/config/i18n/fr.json

```diff
@@ -243,6 +244,11 @@
         "addButton": "+",
         "substractButton": "-"
       },
+      "devices": {
+        "editDeviceFeaturesLabel": "Sélectionnez les appareils que vous voulez afficher ici.",
+        "editNameLabel": "Nom du widget",
+        "editNamePlaceholder": "Enter le nom du widget"
```

Comment:
Typo!
Should be:  "Entrez le nom du widget"

Reply:
Oops :(